.github/workflows/pinot_tests.yml

```diff
@@ -281,4 +273,4 @@ jobs:
         env:
           MAVEN_OPTS: -Xmx2G -DfailIfNoTests=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.wagon.http.retryHandler.count=30 -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false
         run: |
-          mvn clean install -DskipTests -Ppresto-driver -am -B -pl ':presto-pinot-driver' -Djdk.version=8 -T 16 || exit 1
+          mvn clean install -Dmaven.test.skip=true -Ppresto-driver -am -B -pl ':presto-pinot-driver' -Djdk.version=8 -T 16 || exit 1
```

Comment:
`maven.test.skip` implies `skipTests`. The main difference is that `maven.test.skip` does not compile and does not run the tests, while `skipTests` compiles them but do not run them. Apart from getting some cycles, the main reason to skip the compilation is to avoid errors there. I've found two different issues:
- Some plugins do not run with modern jvms
- Some libraries we use have been compiled with java 11
The latter is the the reason why we need to skip compilation here. The environment is here configured to run with Java 8 in order to compile presto, but introduced a library that uses Java 11 (IIRC the library is mockito). We updated these libraries in order to be able to run in Java 20 (in order to have it ready for Java 21).